src/main/java/net/fabricmc/loader/impl/game/minecraft/launchwrapper/FabricTweaker.java

```diff
@@ -118,6 +116,7 @@ public void injectIntoClassLoader(LaunchClassLoader launchClassLoader) {
 			throw new RuntimeException("Could not locate Minecraft: provider locate failed");
 		}
 
+		arguments = null;
```

Comment:
Why null out the arguments? Just because they're not needed anymore?

Reply:
maybe to allow garbage collection?

Reply:
just to trap any accidental reuse